e2e/test/prerequisites/iot_config.cmd_template

```diff
@@ -92,6 +92,10 @@ set CUSTOM_ALLOCATION_POLICY_WEBHOOK=
 @REM Only the hostname is needed, not the full connection string
 set FAR_AWAY_IOTHUB_HOSTNAME=
 
+@REM Storage account integration
+@REM Used for IoT Hub features that integrate with a storage account, e.g. DeviceImport and DeviceExport jobs
+set STORAGE_ACCOUNT_CONNECTION_STRING=
```

Comment:
Introducing a new environment variable. Will need to get this set up for our gates so they can run this test as well.